ui/src/pages/Experiments/index.tsx

```diff
@@ -1,18 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
 
-import { Button, Card, CardContent, Typography } from '@material-ui/core'
+import { Button, Card, CardContent, Drawer, Typography } from '@material-ui/core'
 import AddIcon from '@material-ui/icons/Add'
+
 import PageBar from '../../components/PageBar'
 import ToolBar from '../../components/ToolBar'
 import Container from '../../components/Container'
+import NewExperiment from '../../pages/Experiments/New'
 
 export default function Experiments() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggleDrawer = (isOpen: boolean) => (event: React.MouseEvent | React.KeyboardEvent) => {
```

Comment:
The event params can be removed because we haven't used it currently.

Reply:
ok, I'll remove this.